pkg/apis/logstash/v1alpha1/validations.go

```diff
@@ -86,3 +90,19 @@ func checkSinglePipelineSource(a *Logstash) field.ErrorList {
 
 	return nil
 }
+
+func checkESRefsNamed(l *Logstash) field.ErrorList {
+	var notNamed []string
+	for _, o := range l.Spec.ElasticsearchRefs {
+		if o.ClusterName == "" {
+			notNamed = append(notNamed, o.NamespacedName().String())
+		}
+	}
+	if len(notNamed) > 0 {
+		msg := fmt.Sprintf("when declaring multiple refs all have to be named, missing clusterName on %v", notNamed)
```

Comment:
I think the error message needs to state that `clusterName` is mandatory for any number of elasticsearchRefs, not just multiple

```suggestion
		msg := fmt.Sprintf("clusterName is a mandatory field - missing on %v", notNamed)
```